src/main/java/mekanism/client/ClientProxy.java

```diff
@@ -590,13 +588,14 @@ public void registerBlockRenders() {
             if (type != null) {
                 String resource = "mekanism:" + type.getName();
                 if (type == MachineType.BASIC_FACTORY || type == MachineType.ADVANCED_FACTORY || type == MachineType.ELITE_FACTORY) {
-                    RecipeType recipe = ((ItemBlockMachine) stack.getItem()).getRecipeTypeOrNull(stack);
+                    RecipeType recipe = ((ItemBlockMachine) stack.getItem()).getRecipeType(stack);
                     if (recipe != null) {
                         resource = "mekanism:" + type.getName() + "_" + recipe.getName();
                     }
                 }
                 return machineResources.get(resource);
             }
+            //TODO: This is annotated Nonnull
```

Comment:
Only because of Forge's hamfisted nonnull package annotations - any use of the `ItemMeshDefinition` interface checks for null.

I'd recommend adding an external annotation in IDEA for it